manifests/plugin/pulp.pp

```diff
@@ -55,16 +55,19 @@
     foreman_proxy::settings_file { 'pulp':
       template_path => 'foreman_proxy/plugin/pulp.yml.erb',
       enabled       => $enabled,
+      feature       => 'Pulp',
       listen_on     => $listen_on,
     },
     foreman_proxy::settings_file { 'pulpnode':
       template_path => 'foreman_proxy/plugin/pulpnode.yml.erb',
       enabled       => $pulpnode_enabled,
+      feature       => 'PulpNode',
```

Comment:
Is this supposed to be 'Pulp Node' to match what gets advertised by the proxy?  Or does it really matter/

Reply:
or i guess the proxy just reports 'pulpnode', foreman knows it as 'Pulp Node'

Reply:
It is what https://theforeman.org/api/1.24/apidoc/v2/smart_proxies/show.html returns under `features` because that's what it checks against.

The background is that the `foreman_proxy::register` collects all these features for enabled modules and after the register succeeds, the features are compared. Missing features cause a failure because then Foreman didn't recognize the feature for some reason.

Reply:
Then i think it should be "Pulp Node":
```
{
   "created_at":"2020-03-17 14:48:58 -0400",
   "updated_at":"2020-03-17 14:48:59 -0400",
   "name":"capsule-dev.balmora.example.com",
   "id":2,
   "url":"https://capsule-dev.balmora.example.com:9090",
   "remote_execution_pubkey":null,
   "download_policy":"on_demand",
   "supported_pulp_types":{
      "pulp2":{
         "supported_types":[
            "ostree",
            "deb",
            "puppet"
         ]
      },
      "pulp3":{
         "supported_types":[
            "file",
            "yum"
         ],
         "overriden_to_pulp2":[
            "docker"
         ]
      }
   },
   "features":[
      {
         "capabilities":[

         ],
         "name":"Pulp Node",
         "id":3
      },
      {
         "capabilities":[

         ],
         "name":"Templates",
         "id":1
      },
      {
         "capabilities":[

         ],
         "name":"TFTP",
         "id":5
      },
      {
         "capabilities":[

         ],
         "name":"Puppet CA",
         "id":9
      },
      {
         "capabilities":[

         ],
         "name":"Puppet",
         "id":8
      },
      {
         "capabilities":[

         ],
         "name":"Logs",
         "id":13
      },
      {
         "capabilities":[

         ],
         "name":"HTTPBoot",
         "id":14
      },
      {
         "capabilities":[
            "pulp_2to3_migration",
            "pulp_ansible",
            "pulp_container",
            "pulp_file",
            "pulp_rpm",
            "pulpcore"
         ],
         "name":"Pulpcore",
         "id":4
      }
   ],
   "locations":[
      {
         "id":2,
         "name":"Default Location",
         "title":"Default Location",
         "description":null
      }
   ],
   "organizations":[
      {
         "id":1,
         "name":"Default Organization",
         "title":"Default Organization",
         "description":null
      }
   ]
}
```

Reply:
Thanks! Will update.